PF/PF-Assignment-1/Shopping.java

```diff
@@ -0,0 +1,132 @@
+package sc;
```

Comment:
Few things to remember:
1. Improve the indentation
2. Create a separate file for each class
3. Remove unwanted code while committing 
4. Add comments to the code